.github/workflows/ci.yaml

```diff
@@ -29,16 +29,18 @@ jobs:
       with:
         fetch-depth: 0
 
+    - name: Install poetry
+      run: pipx install poetry
```

Comment:
should be "pip" instead of pipx?

Reply:
pipx is just a handy tool to install poetry
https://python-poetry.org/docs/#installation